### PR TITLE
fix(nginx.conf): use variables instead of upstream directive

### PR DESCRIPTION
**Pull request by pkheav**

### CODE REVIEW

**Comment by pkheav** [comment]
nginx.conf file copied over from here https://github.com/place-labs/partner-environment/pull/45

File: config/nginx.conf [hunk 1]
```diff
@@ -9,7 +8,6 @@ events {
 
 env JWT_PUBLIC;
 env INFLUX_API_KEY;
-env HTTPS_REDIRECT_PORT;
```
Comment (w-le) [inline config/nginx.conf:12]: add this back

File: config/nginx.conf [hunk 2]
```diff
@@ -19,31 +17,14 @@ http {
...
-  }
+  sendfile    on;
+  tcp_nopush  on;
+  gzip        on;
 
   server {
     listen 80 default_server;
     server_name _;
-    set_by_lua $https_redirect_port 'return os.getenv("HTTPS_REDIRECT_PORT")';
-    return 301 https://$host:$https_redirect_port$request_uri;
```
Comment (w-le) [inline config/nginx.conf:46]: so that this can work,
it fixes somethign related to have ports as part of the uri

Comment (w-le) [reply config/nginx.conf:46]: this was it: https://github.com/PlaceOS/nginx/pull/2

let's keep it

**Review by w-le — COMMENTED** [review]
should work other than the 2 comments below red https_redirect_port

File: config/nginx.conf (hunk 2 above)
Comment (pkheav) [reply config/nginx.conf]: thanks Will, I've undid both those changes

**Review by w-le — APPROVED** [review]

**Review by caspiano — APPROVED** [review]

**Comment by pkheav** [comment]
Thanks for approving guys, I don't actually have write access so can't merge otherwise I would just do so myself.

**Pull request merged**